test/product_factory.rb

```diff
@@ -36,6 +36,7 @@ def self.create_product(attrs = {})
     product["description"] = attrs["description"] || "SUSE Linux Enterprise #{product_name}."
     product["display_name"] = attrs["display_name"] || "SUSE Linux Enterprise #{product_name}"
     product["download_size"] = attrs["download_size"] || 0
+    # default: 2024-10-31
     product["eol"] = attrs["eol"] || 1730332800
```

Comment:
np: I know it's not part of your PR, but I'm curious about the meaning of `1730332800`. A constant with a self-descriptive name would be nice. But we can move on with this PR and leave it for later.